src/dataSources/Null.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var Base = require('../lib/Base');
+
+var Null = Base.extend('Null', {
+    isNullObject: function() {
+        return true;
+    },
+    getProperty: function getProperty(propName) {
```

Comment:
Copy in the other methods from hyper-analytics/js/Base.js so this file can serve as a base for any data source.